keras_cv/bounding_box/__init__.py

```diff
@@ -23,8 +23,8 @@
 from keras_cv.bounding_box.formats import YXYX
 from keras_cv.bounding_box.iou import compute_iou
 from keras_cv.bounding_box.pad_batch_to_shape import pad_batch_to_shape
-from keras_cv.bounding_box.utils import add_class_id
+from keras_cv.bounding_box.to_dense import to_dense
+from keras_cv.bounding_box.to_ragged import to_ragged
```

Comment:
Same comments above -- prefer not to add new APIs before 0.4

Reply:
I would view this as a substitution of an API (lines 28-29) rather than an addition of an API:

see:
```
from keras_cv.bounding_box.utils import filter_sentinels
from keras_cv.bounding_box.utils import pad_with_sentinels
```

has been removed, these replace it.

Reply:
can we wait until we move everything to a stable API to worry too much about this?  We will need to use this in training scripts too anyways, so I'd like to avoid the effort of changing all of the imports + replacing the statements if I'm just going to undo it in the next PR.